shunting-yard-replicator/src/main/java/com/expediagroup/shuntingyard/replicator/exec/conf/OrphanedDataStrategyConfig.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (C) 2016-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.shuntingyard.replicator.exec.conf;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import com.hotels.bdp.circustrain.api.conf.OrphanedDataStrategy;
+
+@Configuration
+@ConfigurationProperties
```

Comment:
Doesn't this need a prefix "orphaned-data-strategy" to get bound to the value from the YAML?

Reply:
I don't _think_ so because in this case there are no "sub" properties. Spring is clever enough to know to map the yaml property "orphaned-data-strategy" to the field in here called "orphanedDataStrategy". I've tested this out on EMR doing an actual replication and it works. Shame there's no integration tests in SY like in CT to test out different config settings.

Reply:
I think can be separate we already have an issue: https://github.com/ExpediaGroup/shunting-yard/issues/5